app_test.go

```diff
@@ -477,3 +478,260 @@ func TestGetAppNoTarget(t *testing.T) {
 		t.Fatal("unexpected <nil> error")
 	}
 }
+
+func TestSetConfig(t *testing.T) {
+	fakeServer := newFakeServer(t)
+	defer fakeServer.stop()
+	fakeServer.prepareResponse(preparedResponse{
+		code:    http.StatusOK,
+		method:  "POST",
+		path:    "/apps/proj1-prod/env",
+		payload: []byte("{}"),
+	})
```

Comment:
It sounds good to do it like that. Since there is even the tsuru-bootstrap project, there could even be a link to it in documentation. The only thing missing would be a server responding to tranor's config.json. However, that could be setup as an app inside tsuru-bootstrap. This could also be documented.

I believe people outside EF can run it perfectly without depending on any internal thing. So I like the idea :)